templates/cli/lib/commands/deploy.js.twig

```diff
@@ -593,6 +599,80 @@ const deployCollection = async ({ all, yes } = {}) => {
     }
 }
 
+const deploySettings = async ({ yes } = {}) => {
```

Comment:
Should we seprate these settings into individual deploys? Might be confusing as these are independent settings?

Reply:
We currently apply:
- OAuth enable/disable
- Services enable/disable
- Max users limit

I can see how it would be useful to only specify one of them to deploy. I can implement a similar passion to what we do in `deploy functions` where you either select ones to deploy manually, or do `--all` to auto-pick all of them. Sounds good?

Reply:
yes that sounds good, otherwise people might be confused as to what settings will be deployed, initialized